src/solver/simulation/include/antares/solver/simulation/sim_structure_probleme_economique.h

```diff
@@ -182,7 +182,8 @@ struct PROPERTIES
     bool penalizeVariationInjection;
 
     std::shared_ptr<Antares::Data::ShortTermStorage::Series> series;
-    std::vector<Antares::Data::ShortTermStorage::AdditionalConstraints> additionalConstraints;
+    std::vector<std::shared_ptr<Antares::Data::ShortTermStorage::AdditionalConstraints>>
```

Comment:
A bit below, we use *::ShortTermStorage::**, so here (line 185), do we need **Antares::Data::** and be that thorough ?

Reply:
I added the full namespace above